mergebot_backend/mergebot_poller.py

```
@@ -57,12 +57,16 @@ def get_authorized_users(self):
         Returns:
             Map of GitHub username to ASF ID of committers for this project.
         """
-        groups_json = requests.get(
-            'https://people.apache.org/public/public_ldap_groups.json')
-        groups_json.raise_for_status()
-        committers_json = requests.get(
-            'https://gitbox.apache.org/setup/ghmap.json')
-        committers_json.raise_for_status()
+        try:
+            groups_json = requests.get(
+                'https://people.apache.org/public/public_ldap_groups.json')
+            groups_json.raise_for_status()
+            committers_json = requests.get(
+                'https://gitbox.apache.org/setup/ghmap.json')
+            committers_json.raise_for_status()
+        except Exception as e:
+            self.l.error("Failed to retrieve authorized users: {err}; using old"
+                         " authorized user list.".format(err=e))
```

should this re-raise or?

---

This should return here; we want to swallow transient network errors.